src/MensaWidget/MWCanteen.class.st

```diff
@@ -68,14 +86,18 @@ MWCanteen >> addMealsTo: aCanteenMorph [
 
 {
 	#category : #display,
-	#'squeak_changestamp' : 'Oscar 6/14/2024 09:01'
+	#'squeak_changestamp' : 'Louis Christ 6/18/2024 12:00'
 }
 MWCanteen >> createNameMorph [
 
-	^ TextMorph new 
-		contents: self name;
-		readOnly: true; 
-		textStyle: TextStyle default;
+	^ AlignmentMorph newRow
+		color: MWColorTheme transparentColor;
+		vResizing: #shrinkWrap;
+		hResizing: #spaceFill;
+		addMorph: (StringMorph contents: self class currency);
+		addMorph: self getSpacerMorph;
+		addMorph: (StringMorph contents: self name);
+		addMorph: self getSpacerMorph;
 		yourself
```

Comment:
Gievn this code an empty canteen would also show the EUR currency. Can we live with that or do i have to make the empty canteen be a nil object (subclass of the canteen) and overwrite this method then?

Reply:
Me personally I can live with that

Reply:
Me too